src/builder.rs

```diff
@@ -2410,7 +2410,36 @@ impl<'ctx> Builder<'ctx> {
 
         unsafe { Ok(T::new(value)) }
     }
-
+    ///creates integer subtraction given `lhs` and `rhs`. It returns either an `IntValue` or `BuilderError`
+    ///```rust,no_run
+    ///fn int_subtraction(lhs: i8, rhs: i8) -> i8{
```

Comment:
nit: can you please fix spacing after the slashes, and capitalize the C in creates

Reply:
done